docs/design/coreclr/botr/readytorun-format.md

```diff
@@ -797,6 +797,7 @@ enum ReadyToRunHelper
     READYTORUN_HELPER_GenericGcTlsBase          = 0x66,
     READYTORUN_HELPER_GenericNonGcTlsBase       = 0x67,
     READYTORUN_HELPER_VirtualFuncPtr            = 0x68,
+    READYTORUN_HELPER_SimpleIsInstanceOf        = 0x69,
```

Comment:
The semantics of this helper are coupled with exception handling. The name should reflect it.

`IsInstanceOfException` or something like that.